### PR TITLE
Update reason reporting in controller

### PR DESCRIPTION
**Pull request by jiangyaoguo**
Now Reason field must be in CamelCase. Update all reason feild in controller in CamelCase format.

### CODE REVIEW

**Comment by jiangyaoguo** [comment]
@bgrant0607 @vishh

File: pkg/controller/node/nodecontroller.go [hunk 1]
```diff
@@ -524,7 +525,8 @@ func (nc *NodeController) tryUpdateNodeStatus(node *api.Node) (time.Duration, ap
 				node.Name, nc.now().Time.Sub(savedNodeStatus.probeTimestamp.Time), lastReadyCondition)
 			if lastReadyCondition.Status != api.ConditionUnknown {
 				readyCondition.Status = api.ConditionUnknown
-				readyCondition.Reason = fmt.Sprintf("Kubelet stopped posting node status.")
+				readyCondition.Reason = "NodeStatusStopUpdated"
```
Comment (vishh) [inline pkg/controller/node/nodecontroller.go]: nit: Can this be `NodeStatusUnknown`?

**Comment by jiangyaoguo** [comment]
@vishh  Changed to "NodeStatusUnknown".

File: pkg/controller/service/servicecontroller.go [hunk 2]
```diff
@@ -671,7 +671,7 @@ func (s *ServiceController) lockedUpdateLoadBalancerHosts(service *api.Service,
 	}
 
 	message := "error updating loadbalancer with new hosts: " + err.Error()
-	s.eventRecorder.Event(service, "updating loadbalancer failed", message)
+	s.eventRecorder.Event(service, "UpdatingLoadBalancerFailed", message)
```
Comment (vishh) [inline pkg/controller/service/servicecontroller.go]: nit: `LoadBalancerUpdateFailed`

**Comment by vishh** [comment]
Just one nit. Otherwise LGTM. Thanks @jiangyaoguo

**Comment by jiangyaoguo** [comment]
@vishh  Changed.

**Comment by vishh** [comment]
LGTM. Thanks for the cleanup @jiangyaoguo!

**Pull request merged**